src/main/groovy/edu/oregonstate/mist/core/PersonObject.groovy

```diff
@@ -0,0 +1,23 @@
+package edu.oregonstate.mist.core
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+
+class PersonObject {
+    @JsonIgnore
+    String osuID
+    String lastName
+    String homePhone
+    String alternatePhone
+    String osuUID
+    Date birthDate
```

Comment:
For all the dates, when I did some simple initial testing, it looks like the date format happened to adhere to [ISO8601 standards](https://xml2rfc.tools.ietf.org/public/rfc/html/rfc3339.html#anchor14) for a "full-date", which is what we want our API to produce. However, I'm thinking we should explicitly state declare this format instead of assuming that our API will produce it. [Here's a similar method used for the locations-api](https://github.com/osu-mist/locations-api/blob/develop/src/main/groovy/edu/oregonstate/mist/locations/core/DayOpenHours.groovy#L9). I'm thinking we want to change all date fields for the persons API (for all endpoints that have dates) to declare the intended format in the code.

Another thought I had is whether we want to produce dates in UTC, like we do for the locations api and some other APIs. There are a few complications to this:
- In Banner, most date fields are of the `Date` datatype, which has no concept of timezone, so we'd have to assume the timezone of all dates (PST or PDT depending on the date). This would be a configuration parameter. I did a similar thing with the OSU Events API (not the message api) except I was dealing with a different backend system (localist rather than banner/oracle) https://github.com/osu-mist/osu-events-api/blob/develop/src/main/groovy/edu/oregonstate/mist/osuevents/resources/EventsResource.groovy#L45-L50
- I'm not even sure if banner keeps a record of time for fields like birth dates, and start and end dates of jobs. If they don't, or if the times aren't reliable, I don't know if we should convert them to UTC. For example, if someone's job start date is January 1st 2018 at 6:00 pm, the UTC date for this would be January 2nd. 

What are your thoughts @tsoliangwu0130 ?